### DISCUSSION TITLE
Per Path RTSPS otpion

### DISCUSSION THREAD

**Discussion by t3therdev** [bluenviron/mediamtx#365] [General]
Right now we are using RTSPSS and it works great.

Our question is that for RTSPS support, its a global setting, so all streams would be accessible in both RTSP and RTSPS mode.

Would it be possible to make that setting per path? So certain streams would reply on RTSPS and others on both?

**Comment by aler9** [comment]
Hello, the problem of the actual RTSPS setting is that it toggles the RTSPS listener and the RTSP listener, therefore it can't be moved in `paths` - another setting should be created but i think it would be confusing for users.

[a comment by an automated account is not shown]
